src/runtime/runtime-test.js

```diff
@@ -1515,4 +1486,58 @@ describes.realWin('ConfiguredRuntime', {}, env => {
   it('should return events manager', () => {
     expect(runtime.eventManager() instanceof ClientEventManager).to.be.true;
   });
+
+  describe('configPromise', () => {
+    let resolveConfig;
+    let rejectConfig;
+    let eventManager;
+    let configPromise;
+
+    const event = {
+      eventType: AnalyticsEvent.IMPRESSION_PAYWALL,
+      eventOriginator: EventOriginator.SWG_CLIENT,
+      isFromUserAction: true,
+      additionalParameters: null,
+    };
+
+    beforeEach(() => {
+      activityResultCallbacks = {};
```

Comment:
is this getting used?

Reply:
There is an error thrown if I do not include this statement.